samples/RemoteAuth/Forms/FormsAuthCore/Program.cs

```diff
@@ -11,7 +11,9 @@
         .Configure(options =>
         {
             options.RemoteAppUrl = new(builder.Configuration["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"]);
-            options.ApiKey = "FormsAuthSampleKey";
+
+            // Do not re-use this ApiKey; every solution should use a unique ApiKey
+            options.ApiKey = builder.Configuration["RemoteAppApiKey"];
```

Comment:
as above, can we put this in the config file and not here?

Reply:
Thanks for the reminder. I didn't have the comment there originally because I still think of comments as belonging in code rather than json, but ASP.NET Core is fine with comments in appsettings.json, so I've moved these.

Reply:
> ASP.NET Core is fine with comments in appsettings.json

Good to know! I wasn't sure if it would yell at you if you put comments there